tools/packaging/build/agent-enclave-bundle/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM ubuntu:20.04 as builder
 
 ARG RATS_TLS_REV=5de6fc308f2a18f4105429451c85e6173b42ab9d
+ARG KBC=cc-kbc
```

Comment:
Can you verify that the payload images we publish continue to work, e.g., in the operator CI where we test with the SIM environment and simplekbc.

Also, if we default to this, can we stop building RATS-TLS to the occlum instance

Reply:
> Can you verify that the payload images we publish continue to work, e.g., in the operator CI where we test with the SIM environment and simplekbc.

Sorry, I am not familiar with this. Could you help with this? I've updated the payload publishment CI in the following commits.

> Also, if we default to this, can we stop building RATS-TLS to the occlum instance
Good point! I've update the Dockerfile to have a conditional build depending on env `KBC`